pkg/controllers/consolidation/types.go

```diff
@@ -20,70 +20,99 @@ import (
 
 	v1 "k8s.io/api/core/v1"
 
+	"github.com/aws/karpenter/pkg/apis/provisioning/v1alpha5"
 	"github.com/aws/karpenter/pkg/controllers/provisioning/scheduling"
 )
 
-// ProcessResult is used to indicate the result of consolidating so we can optimize by not trying to consolidate if
+// LifecycleResult is used to indicate the action of consolidating so we can optimize by not trying to consolidate if
 // we were unable to consolidate the cluster and it hasn't changed state with respect to pods/nodes.
-type ProcessResult byte
+type LifecycleResult byte
```

Comment:
Nick, Bill and I discussed this name, and we ended up really liking the following for how we talk about this in the docs
```suggestion
type DeprovisioningResult byte
```